.github/pull_request_template.md

```diff
@@ -0,0 +1,57 @@
+# Description
+<!--
+  This is a template to add as many information as possible to the pull request, to help the reviewer and as a checklist for you. Points to remember are set in the comments, please read and keep them in mind:
+
+    - Code should be self-explanatory and share your knowledge with others
+    - Document code that is not self-explanatory
+    - Think about bugs and keep security in mind
+    - Write tests (Unit and Integration), also for error cases
+    - Main logic should hidden behind the api, never trust the client
+    - Visible changes should be discussed with the UX-Team from the begining of development; they also have to accept them at the end
+    - Keep the changelog up-to-date
+    - Leave the code cleaner than you found it. Remove unnecessary lines. Listen to the linter.
+-->
+
+## Links to Tickets or other pull requests
+<!--
+Base links to copy
+- https://github.com/dBildungsplattform/dbildungs-iam-server/pulls
+- https://ticketsystem.dbildungscloud.de/browse/SPSH-???
+- https://ticketsystem.dbildungscloud.de/browse/EW-???
+-->
+
+## Changes
+<!--
+  What will the PR change?
+  Why are the changes required?
+  Short notice if a ticket exists, more detailed if not
+-->
+
+## Datasecurity
+<!--
+  Notice about:
+  - model changes
+  - logging of user data
+  - right changes
+  - and other user data stuff
+  If you are not sure if it is relevant, take a look at confluence or ask the data-security team.
+-->
+
+## Deployment
+<!--
+  - Keep in mind to change seed data, if changes are done on migration scripts.
+  - What is required for deployment after changes on infrastructure
+  - Explain Environment variables like FEATURE_XY=true

```

Comment:
Leave a template for those variables.

Include a comment explaining the change and include possible values if they're limited

Reply:
Since we won't be using Environment Variables this point is unnecessary imo. Rather change it to include changes to the config schema that need to be adressed with helm charts

Reply:
Yes. We don't want to use env vars.